tests/conftest.py

```diff
@@ -1,21 +1,27 @@
+from asyncio import AbstractEventLoop
+from typing import AsyncGenerator, cast
+
 import aiohttp
 import pytest
+from pytest_asyncio.plugin import SubRequest
 
 
 @pytest.fixture(
-    scope="session", params=[True, False], ids=["debug:true", "debug:false"]
+    scope="session",
+    params=[True, False],
+    ids=["debug:true", "debug:false"],
 )
-def debug(request):
-    return request.param
+def debug(request: SubRequest) -> bool:
```

Comment:
Isn't this `pytest.FixtureRequest`?
https://github.com/pytest-dev/pytest/blob/407d9841428f037e0ada71f7ca028f7d1171c6db/src/_pytest/fixtures.py#L340-L344

Reply:
From FixtureRequest:

        # -`request.param` is only defined in parametrized fixtures, and will raise
        #   AttributeError otherwise. Python typing has no notion of "undefined", so
        #   this cannot be reflected in the type.
        # - Technically `param` is only (possibly) defined on SubRequest, not
        #   FixtureRequest, but the typing of that is still in flux so this cheats.
        # - In the future we might consider using a generic for the param type, but
        #   for now just using Any.

Reply:
Then `pytest.SubRequest`?

Reply:
> Then `pytest.SubRequest`?

Both is okay, since both refers to `Any`